README.md

```diff
@@ -42,11 +42,13 @@ examples are located in the [scripts/](./scripts/) folder.
 
 Load script | Key topics
 ----------- | ----------
-[load-csv-file](./scripts/load-csv-file) | Loading data from a CSV file.
-[load-xlsx-file](./scripts/load-xlsx-file) | Loading data from an Excel file.
-[load-inline-table](./scripts/load-inline-table) | Loading an inline table 
-[load-subset-of-fields](./scripts/load-subset-of-fields) | Loading a subset of fields.
-[rename-fields](./scripts/rename-fields) | Renaming fields.
-[load-field-functions](./scripts/load-field-functions)
-[concat-tables-auto](./scripts/concat-tables) | Automatic concatenation into one table.
-[concat-tables-forced](./scripts/concat-tables) | Forced concatenation into one table.
+[load-csv-file](./scripts/load-csv-file) | <ul><li>Loading data from a CSV file.</ul>
```

Comment:
Do lists render ok? Let's check. If not, I'll revert.

Reply:
Yes they do :)